src/features/emails/utils/editorjsBlocksToZetkinBlocks.spec.ts

```diff
@@ -0,0 +1,119 @@
+import editorjsBlocksToZetkinBlocks from './editorjsBlocksToZetkinBlocks';
+import { BLOCK_TYPES, BlockKind, InlineNodeKind } from '../types';
+
+describe('editorjsBlocksToZetkinBlocks()', () => {
+  it('does nothing when passed an empty array', () => {
+    const zetkinBlocks = editorjsBlocksToZetkinBlocks([]);
+
+    expect(zetkinBlocks).toHaveLength(0);
+  });
+
+  it('converts Button block', () => {
+    const zetkinBlocks = editorjsBlocksToZetkinBlocks([
+      {
+        data: {
+          buttonText: 'Click me!',
+          tag: 'abcdefgh',
+          url: 'http://www.zetkin.org',
+        },
+        type: BLOCK_TYPES.BUTTON,
+      },
```

Comment:
I was surprised at first that you didn't have `id` in these blocks, but then I realized when looking at the types in editorjs that `id` is optional. I would still add `id` to a block in at least one of these tests, to be able to assert that the ID does not make it out of `editorjsBlocksToZetkinBlocks()`.